tests/test_landings.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import json
+import pytest
+
+from landoapi.models.storage import db as _db
+from landoapi.models.landing import (Landing, TRANSPLANT_JOB_STARTED)
+
+from tests.canned_responses.phabricator.revisions import *
+from tests.canned_responses.lando_api.revisions import *
+from tests.canned_responses.lando_api.landings import *
+
+
+@pytest.fixture
+def db(app):
+    """Monkeypatch environment variables that we'd get running under docker."""
+    with app.app_context():
+        _db.init_app(app)
+        _db.create_all()
+        # Yield the connexion app so other fixtures can get the app instance
+        # we just created.
+        yield _db
+        _db.session.remove()
+        _db.drop_all()
+
+
+def test_landing_revision(db, client, phabfactory):
```

Comment:
This test name should be something like `test_landing_request_is_saved_to_database()`.